test/option/fixture/taskmaster_expected_new_parallel.txt

```diff
@@ -1,6 +1,6 @@
 Job.NewParallel._work(): [Thread:XXXXX] Gained exclusive access
 Job.NewParallel._work(): [Thread:XXXXX] Starting search
-Job.NewParallel._work(): [Thread:XXXXX] Found {len(results_queue)} completed tasks to process
+Job.NewParallel._work(): [Thread:XXXXX] Found 0 completed tasks to process
```

Comment:
This is sufficient to get `test/option/taskmastertrace.py` to pass with the fixed f-string in Job.py.